banners/english/WMDE_FR_2025_Desktop_EN_01/content/BannerSlides.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+	<KeenSliderSlide :is-current="currentSlide === 0">
+		<p class="headline">
+			<strong>
```

Comment:
We should try to avoid unsemantic (i.e. formatting-only) markup (like `<strong>` tags for a whole paragraph) in our code.

We have a `.headline` style in the Slider CSS definitions in our theme. Please remove the `<strong>` tag in the Slides.

For the non-slider text, please add the `.headline` class to the `p` tag and define a `.headline` style inside the `&-message` rule in `MainBanner.scss`

Reply:
I wouldn't necessarily say its unsemantic here. It conveys the meaning that that part of text is emphasised more than the others. What we're doing here is basically `<p class="strong">` only we called it `.headline`. Using a strong tag also means we have one less style to maintain.

Screen reader users don't like emphasis being announced though, and `.headline` is already part of our themes so lets keep it.